examples/.env

```diff
@@ -0,0 +1,7 @@
+MARIADB_ROOT_PASSWORD=secret
+MYSQL_INITDB_SKIP_TZINFO=Y
```

Comment:
There is [MARIADB_INITDB_SKIP_TZINFO](https://mariadb.com/kb/en/mariadb-server-docker-official-image-environment-variables/#mariadb_initdb_skip_tzinfo-mysql_initdb_skip_tzinfo), but is there really a need to exclude tz init? Generally like to keep examples small.